subsys/net/ip/Kconfig

```diff
@@ -536,10 +537,20 @@ config NET_PKT_TIMESTAMP
 	  example in gPTP which needs to know how long it takes to send
 	  a network packet.
 
+config NET_PKT_TIMESTAMP_THREAD
```

Comment:
If, per the description, this thread is needed only for gPTP, while option name doesn't include reference to it?

Reply:
gPTP needs this support, but timestamping does not need gptp. Currently only gptp uses or needs this one but I do not see a reason to mention gptp in the option name.